### PR TITLE
Mckin 8055 assessment bar ui V4

### PR DESCRIPTION
**Pull request by murad-hubib**
- Added assessment bar in assessment view for V4

> Color related changes will be implemented in MCKA-Theme

Ticket: [https://edx-wiki.atlassian.net/browse/MCKIN-8055](https://edx-wiki.atlassian.net/browse/MCKIN-8055)

@xitij2000 @kaizoku

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @murad-hubib! I've created [OSPR-2558](https://openedx.atlassian.net/browse/OSPR-2558) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).



We can't start reviewing your pull request until you've submitted a     [signed contributor agreement](http://open.edx.org/sites/default/files/wysiwyg/individual-contributor-agreement.pdf)     or indicated your institutional affiliation. Please see the [CONTRIBUTING](https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst)     file for more information.

**Comment by xitij2000** [comment]
@murad-hubib  Could you look into the test failures?

**Comment by viadanna** [comment]
Hello @murad-hubib I was testing this change but the number of attempts does not show in my testing. Do you expect further changes here?
screen shot 2018-08-24 at 13 45 25

**Comment by murad-hubib** [comment]
@viadanna Looks like you didn't set the maximum attempts in studio. By default, it's infinite

**Comment by viadanna** [comment]
My bad @murad-hubib it does show after making sure there's a maximum number of attempts.

Is this PR tackling the mobile design as well?

**Comment by murad-hubib** [comment]
@viadanna No, it doesn't tackle mobile design

**Comment by murad-hubib** [comment]
@xitij2000 All tests have passed. Can you please review and merge?

File: drag_and_drop_v2/public/css/drag_and_drop.css [hunk 1]
```diff
@@ -709,19 +709,69 @@
...
+
 }
 
 .xblock--drag-and-drop .actions-toolbar .action-toolbar-item {
     display: inline-block;
     margin: 10px 0;
 }
 
+.xblock--drag-and-drop .actions-toolbar .submit-answer-button{
+    padding: 0.625em 2.45em;
```
Comment (viadanna) [inline drag_and_drop_v2/public/css/drag_and_drop.css]: It seems the padding breaks the mobile view. Could you reduce it?

screen shot 2018-08-30 at 18 37 09

**Comment by murad-hubib** [comment]
@viadanna I have removed that extra padding I will add in mcka-theme

**Comment by viadanna** [comment]
Good to go 👍

- [x] I tested this works on devstack.
- [x] I tested the accessibility.
- [x] I read through the code.
- [x] I Documentation unnecessary.

@xitij2000 Can you merge this?

**Pull request merged**